example-init-wallet-k8s.sh

```diff
@@ -23,8 +23,8 @@ lndinit gen-password \
   --target=k8s \
   --k8s.base64 \
   --k8s.namespace="${WALLET_SECRET_NAMESPACE}" \
-  --k8s.secret-name="${WALLET_SECRET_NAME}" \
-  --k8s.secret-key-name=walletpassword
+  --k8s.name="${WALLET_SECRET_NAME}" \
```

Comment:
We'd need to update all our charts to make sure the startup script is correct. So I'd strongly suggest not renaming things as external users might also already be using this.

Reply:
✅